src/routes/index.tsx

```diff
@@ -11,6 +12,12 @@ export const paths = {
   root: '/',
   signIn: '/sign-in',
   survey: '/surveys/:id',
+  question: '/surveys/:id/questions',
+};
+
+export const questionPath = (): string => {
```

Comment:
can't it just `questionPath = 'questions'` 🤔

Reply:
@nvminhtue  fixed in [cf449dd](https://github.com/manh-t/react-survey/pull/75/commits/cf449ddc877e55d3fd751f3393ec3d644c023942)